release_notes/topics/18_0_0.adoc

```diff
@@ -1,5 +1,22 @@
 = Highlights
 
+== New Operator tech-preview
+
+With this release, we're introducing a brand new {project_operator} as a tech-preview. Apart from being rewritten from
+scratch, the main user-facing change from the legacy Operator is the used {project_name} distribution – the new Operator
+uses the Quarkus distribution of {project_name}. With that, there are a few breaking changes. For details, incl.
+installation and migration instructions, see the https://www.keycloak.org/guides[Operator related guides].
+
+The link:{operatorRepo_link}[legacy Operator] will receive updates until June 2022 when the {project_name} WildFly
```

Comment:
Is June actually included? Then I'd write until start of July or sth. instead until is a bit ambiguous at times ;)

Reply:
@DGuhr Thank you for your review!

I'm simply aligning it with https://github.com/keycloak/keycloak-documentation/blob/main/release_notes/topics/17_0_0.adoc#quarkus-distribution-is-now-fully-supported. :)
> With that in mind we will continue to support the WildFly distribution until June 2022.